x-pack/filebeat/docs/inputs/input-entity-analytics.asciidoc

```diff
@@ -279,6 +280,14 @@ The client/application ID. Used for authentication. Field is required.
 
 The secret value, used for authentication. Field is required.
 
+[float]
+===== `dataset`
+
+The datasets to collect from the API. This can be one of "all", "users" or "devices",
```

Comment:
```suggestion
The dataset to collect from the API. This can be one of "all", "users" or "devices"
```

Reply:
No, this is intended to be plural and the Oxford comma is required.

Reply:
It was just a thought though.. nvm.